SPECS/ocaml-systemd.spec

```diff
@@ -0,0 +1,90 @@
+%global opt %(test -x %{_bindir}/ocamlopt && echo 1 || echo 0)
+%global debug_package %{nil}
+%global _use_internal_dependency_generator 0
+%global __find_requires /usr/lib/rpm/ocaml-find-requires.sh
+%global __find_provides /usr/lib/rpm/ocaml-find-provides.sh
+
```

Comment:
`_use_internal_dependency_generator`, `__find_requires`, `__find_provides` are not required.   Please remove these lines.